cost/aws/unused_ip_addresses/README.md

```diff
@@ -37,7 +37,6 @@ The following policy actions are taken on any resources found to be out of compl
 ## Prerequisites
 
 - This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
-- billing_center_viewer (note: this role must be applied at the Organization level)
```

Comment:
Are you still getting currency information from Optima? If so, you probably need some role in the Flexera One or legacy RightScale platform.